src/librustdoc/clean/blanket_impl.rs

```diff
@@ -115,12 +122,12 @@ impl<'a, 'tcx> BlanketImplFinder<'a, 'tcx> {
                             ),
                             // FIXME(eddyb) compute both `trait_` and `for_` from
                             // the post-inference `trait_ref`, as it's more accurate.
-                            trait_: Some(trait_ref.clean(cx)),
+                            trait_: clean_trait_ref,
                             for_: ty.clean(cx),
                             items: cx.tcx
                                 .associated_items(impl_def_id)
                                 .in_definition_order()
-                                .map(|x| x.clean(cx))
+                                .map(|x| cx.items_from_blanket.get(&x.def_id).cloned().unwrap_or_else(|| x.clean(cx)))
```

Comment:
You never insert into this map, it's always empty.

Reply:
... I don't know how I could not see that. Thanks a lot!